tsconfig.cjs.json

```diff
@@ -3,9 +3,12 @@
   "compilerOptions": {
     "target": "ES5",
     "module": "CommonJS",
-    "outDir": "./lib/cjs",
+    "outDir": "./lib",
     "sourceMap": false,
     "declaration": true,
     "declarationMap": true
-  }
+  },
+  "include": [
```

Comment:
I do not see the purpose of this change....can you explain? (I think it was better in the base config)

Reply:
This change was necessary because of VSCode. It picks up configuration from *tsconfig.json*, so if `include` was placed there, then error diagnostics wouldn't work properly in non-included *.ts* files like tests (since if `include` is omitted, [`all *.ts* files are included by default](https://www.typescriptlang.org/tsconfig#include)).

AFAIK, unfortunately, there's currently [no way to configure VSCode to use another TS config file than `tsconfig.json`](https://github.com/microsoft/vscode/issues/12463).

Reply:
ok, thanks to clarify it.